packages/babel-helpers/src/helpers/replaceTsImportExt.ts

```diff
@@ -0,0 +1,7 @@
+/* @minVersion 7.25.7 */
+
+export default function replaceTsImportExt(source: string) {
+  return /[\\/]/.test(source)
```

Comment:
Here, it is not sufficient to detect the bare import; consider `@foo/bar.ts` .

Reply:
That _might_ need to be rewritten, if for example you are remapping
```html
<script type="importmap">
  {
    "imports": {
      "@components/": "./ui/components"
    }
  }
</script>
```

We added the exception for "it needs to have a slash" because:
- there are known npm packages with no scope that end in `.ts`
- if you use import maps and have no slash, it means you are explicitly writing the import specifier in your import map

So far nobody ever reported a problem with the `@foo/bar.ts` handling, but if there is a package out there needing it the only way forward is to provide a function option in the config.